hawkbit-ui/src/main/java/org/eclipse/hawkbit/ui/customrenderers/client/renderers/CustomObject.java

```diff
@@ -0,0 +1,47 @@
+/**
+ * Copyright (c) 2015 Bosch Software Innovations GmbH and others.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.eclipse.hawkbit.ui.customrenderers.client.renderers;
+
+import java.io.Serializable;
+
+public class CustomObject implements Serializable {
```

Comment:
Is there maybe a better class name for `CustomObject` more meaningful name what this Object is doing? holding information? `CustomObject` is for me not meaningful enough :)